go/chat/helper.go

```diff
@@ -615,23 +659,46 @@ func (n *newConversationHelper) create(ctx context.Context) (res chat1.Conversat
 	// there is a ton of logic in there to try and present a nice looking menu to help out the
 	// user and such. For the most part, the CLI just uses FindConversationsLocal though, so it
 	// should hopefully just result in a bunch of cache hits on the second invocation.
-	onechatpertlf := true
-	convs, irl, err := FindConversations(ctx, n.G(), n.DebugLabeler, n.ri, n.uid, n.tlfName, n.topicType,
-		n.membersType, n.vis, findConvsTopicName, &onechatpertlf)
-	if err != nil {
-		return res, rl, err
-	}
+
+	convs, irl, err := n.findExisting(ctx, findConvsTopicName)
+
 	// If we find one conversation, then just return it as if we created it.
 	rl = append(rl, irl...)
 	if len(convs) == 1 {
 		n.Debug(ctx, "found previous conversation that matches, returning")
 		return convs[0], rl, err
 	}
 
-	info, err := CtxKeyFinder(ctx, n.G()).Find(ctx, n.tlfName, n.membersType,
-		n.vis == chat1.TLFVisibility_PUBLIC)
+	// if KBFS, return an error. Need to use IMPTEAM now.
+	if n.membersType == chat1.ConversationMembersType_KBFS {
+		// let it slide in devel for tests
+		if n.G().ExternalG().Env.GetRunMode() != libkb.DevelRunMode {
+			return res, rl, errors.New("new KBFS conversations no longer allowed, use IMPTEAM")
```

Comment:
We could just change the membersType to IMPTEAM here.

Reply:
True